tests/gem5/readfile_tests/configs/test_binary.sh

```diff
@@ -0,0 +1,2 @@
+echo "running test binary from readfile_tests"
```

Comment:
Is this test script supposed to execute a binary as well? If not, I'd change the echo statement to 

```suggestion
echo "running test script from readfile_tests"
```
and change the name of the script to `test_script.sh`.